src/main/java/com/purbon/kafka/topology/model/users/Consumer.java

```diff
@@ -18,6 +18,11 @@ public Consumer(String principal) {
     group = Optional.empty();
```

Comment:
@MagnusSmith do you mind updating as well this constructor to reference the more common one downstream where the group is as well set? so i mean there is only one constructor ;-)

Reply:
@purbon just to confirm are you saying to remove the additional constructor `Consumer(String principal, String group)` 
so instead of 
```
Consumer consumer = new Consumer("User:foo", "foo*");
```
to call
```
 Consumer consumer = new Consumer("User:foo");
 consumer.setGroup(Optiona.of("foo*");
```
or do you mean something else?